### ISSUE TITLE
Install wrapper in nested directory

### ISSUE THREAD

**Issue by WezSieTato** [passsy/flutter_wrapper#17]
I have a flutter project that isn’t placed in root repository directory but in `child` dir. Running install command in `child` directory creates `.gitmodules`:
```
[submodule "child/.flutter"]
    path = child/.flutter
    url = https://github.com/flutter/flutter.git
    branch = stable
```
Running any flutter wrapper command in `child` directory shows a warning
```
Warning: Detected detached HEAD. '.flutter' submodule points to a specific commit, not a branch.
You can ignore this warning when you run './flutterw' for the first time after cloning the repository.
fatal: branch name required
usage: grep [-abcDEFGHhIiJLlmnOoqRSsUVvwxZ] [-A num] [-B num] [-C[num]]
    [-e pattern] [-f file] [--binary-files=value] [--color=when]
    [--context[=num]] [--directories=action] [--label] [--line-buffered]
    [--null] [pattern] [file ...]
Warning: Commit cc949a8e8b9cf394b9290a8e80f87af3e207dce5 in submodule '.flutter' is not part of the branch '' as defined in .gitmodules
You have to manually switch to a valid branch (i.e. './flutterw channel stable') or './flutterw upgrade' will fail
Alternatively change the branch '' in .gitmodules to the correct upstream branch (one of stable|beta|dev).

```
Warning disappears when I run `./flutterw channel stable` but then new `.gitmodules` is created in `child` directory with content:
```
[submodule ".flutter"]
    branch = stable

```
Now I have two `.gitmodules` and as I understand git `child/.gitmodules` has no effect on git and is only used to read branch by flutterw script. Is this the expected behaviour?

**Comment by Kisty** [comment]
Can this be closed now?

**Comment by bottee** [comment]
I'm also waiting for this issue to be solved. Is there still an issue with the [commit](https://github.com/passsy/flutter_wrapper/commit/f6af9793fa7349f858ae8d5c369bde3ca0fa16d3)?
It would be great to have this working!

**Comment by dustin-graham** [comment]
@passsy It looks like you fixed this last November, but you didn't tag the commit and so the install script doesn't download your latest wrapper code.

**Comment by passsy** [comment]
This fix has been released in [v1.2.0](https://github.com/passsy/flutter_wrapper/releases/tag/v1.2.0). Please upgrade via 

```
sh -c "$(curl -fsSL https://raw.githubusercontent.com/passsy/flutter_wrapper/master/install.sh)"
```
